modules/vlanlist.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * LMS version 1.11-git
+ *
+ *  (C) Copyright 2001-2020 LMS Developers
+ *
+ *  Please, see the doc/AUTHORS for more information about authors!
+ *
+ *  This program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License Version 2 as
+ *  published by the Free Software Foundation.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307,
+ *  USA.
+ *
+ *  $Id$
+ */
+
+if (!ConfigHelper::checkPrivilege('network_management')) {
+    access_denied();
+}
```

Comment:
To jest niepotrzebne jeśli masz w accesstable.php.

Reply:
Usunięty zduplikowany kod - https://github.com/lmsgit/lms/pull/1824/commits/94b0e87bb73eb6ab8244664070b75f2f6c1c603b.